src/SprykerEco/Zed/AmazonQuicksight/Business/AmazonQuicksightFacade.php

```diff
@@ -70,4 +72,61 @@ public function expandAnalyticsCollectionWithQuicksightAnalytics(
             ->createAnalyticsExpander()
             ->expandAnalyticsCollectionWithQuicksightAnalytics($analyticsRequestTransfer, $analyticsCollectionTransfer);
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\UserCollectionResponseTransfer $userCollectionResponseTransfer
+     *
+     * @return \Generated\Shared\Transfer\UserCollectionResponseTransfer
+     */
+    public function deleteQuicksightUsersByUserCollectionResponse(
+        UserCollectionResponseTransfer $userCollectionResponseTransfer
+    ): UserCollectionResponseTransfer {
+        return $this->getFactory()
+            ->createQuicksightUserDeleter()
+            ->deleteQuicksightUsersByUserCollectionResponse($userCollectionResponseTransfer);
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @return \Generated\Shared\Transfer\QuicksightListUsersResponseTransfer
+     */
+    public function getQuicksightUsersList(): QuicksightListUsersResponseTransfer
```

Comment:
Is it needed? It doesn't even have an interface

Reply:
Leftover from testing, removed